app/src/main/java/org/simple/clinic/settings/changelanguage/ChangeLanguageModel.kt

```diff
@@ -13,9 +13,18 @@ data class ChangeLanguageModel(
 
   fun withCurrentLanguage(currentLanguage: Language): ChangeLanguageModel {
     return copy(currentLanguage = currentLanguage)
+        .coerceUserSelectedLanguage()
```

Comment:
Do we have to do this? Why not leave the `currentLanguage` property to null, unless the user selects one? If this has to go on the UI, we could make the view renderer handle it.

Reply:
Discussed offline.